doc/cookbook.rst

```diff
@@ -22,28 +22,31 @@ following is for one instrument in one working directory.
 1. Prepare your data
 
   - Identify folder(s) with raw images
-  - The raw images can be gzip compressed
+  - The raw images can be gzip compressed although the Python FITS reader works much more slowly on gzipped files
 
-2. Run the pypit_setup script
+2. Run the :ref:`pypit_setup` script one of the PYPIT-approved :doc:`instruments` (e.g. lris_blue, kast_red)
 
-    - Generates the instrument PYPIT reduction file (:doc:`pypit_file`)
-    - Generates the instrument .setups file (:doc:`setups`)
-    - Generates a custom PYPIT reduction file for each setup
-    - Inspect the .setups file to confirm the instrument configurations
-    - Modify the data and spect blocks in master .pypit file, as needed
-    - If changes were made, rerun pypit_setup
+  - Generates the instrument PYPIT reduction file [not used further]
+  - Generates the instrument .setups file (:doc:`setup`)
+  - Generates a custom PYPIT reduction file for each setup
+  - Scan the output WARNING messages which will indicate insufficient calibration files (e.g. missing arc frames)
 
-3. Isolate and modify the custom PYPIT reduction files (:doc:`pypit_file`)
+3. Inspect the :ref:`setups-file` and :ref:`sorted-file` to confirm the instrument configurations
+  - Modify the data and spect blocks in the instrument PYPIT file, as needed (see :ref:`setup-modifications`)
 
-  - Copy/move the PYPIT reduction file for a given setup into its own folder
+    - If changes were made, rerun :ref:`pypit_setup`
+
+4. Isolate and modify the custom PYPIT reduction files (:doc:`pypit_file`)
+
+  - One should copy/move the PYPIT reduction file for a given setup into its own folder
```

Comment:
I like this suggestion and am going to implement it now.  
It will now require that one run pypit_setup twice (or 
just once if you are feeling lucky).